tst/broken/quick/ProjSubfield3.tst

```diff
@@ -4,7 +4,4 @@ gap> gens := List(GeneratorsOfGroup(h),y->y^x);;
 gap> gens[1] := gens[1] * Z(3^4);;
 gap> gens[2] := gens[2] * Z(3^4)^17;;
 gap> g := GroupWithGenerators(gens);;
-
-# Remark: The *4 comes from the fact that by blowing up we lose a factor
-#         of 4 in scalars, they are now in the projective group.
```

Comment:
(Looking at commit `tst: move broken test ProjSubfield3.tst`):

How has this test broken? In the `master` branch, this test runs fine.

Reply:
It passes very often. But if I run it in a session in a while loop it breaks after maybe a second. It also did break on travis at least once, see e.g. older travis runs for #161.